server/models/Prepaid.coffee

```diff
@@ -47,6 +47,13 @@ PrepaidSchema.methods.canReplaceUserPrepaid = (otherPrepaid) ->
 PrepaidSchema.methods.canBeUsedBy = (userID) ->
   @get('creator').equals(userID) or _.find(@get('joiners'), (joiner) -> joiner.userID.equals(userID) )
 
+PrepaidSchema.methods.wasStudentEnrolledByTeacher = (userID, teacherID) ->
```

Comment:
Does this need a course type check similar to revoke?

Reply:
Ah, yup, good catch.